lib/al/Library/Screen/ScreenFunction.h

```diff
@@ -0,0 +1,107 @@
+void calcWorldPosFromLayoutPos(sead::Vector3f* output, const al::SceneCameraInfo*, const sead::Vector2f, const sead::Vector3f, int);
+void calcWorldPosFromLayoutPosSub(sead::Vector3f* output, const al::IUseCamera*, const sead::Vector2f, float);
+void calcWorldPosFromLayoutPosSub(sead::Vector3f* output, const al::IUseCamera*, const sead::Vector2f, const sead::Vector3f);
+
+void calcScreenPosFromWorldPos(sead::Vector2f* output, const al::IUseCamera*, const sead::Vector3f&);
+void calcScreenPosFromWorldPosSub(sead::Vector2f* output, const al::IUseCamera*, const sead::Vector3f&);
+void calcScreenPosFromLayoutPos(sead::Vector2f* output, const sead::Vector2f&);
+float calcScreenRadiusFromWorldRadius(const sead::Vector3f&, const al::IUseCamera*, float);
+float calcScreenRadiusFromWorldRadiusSub(const sead::Vector3f&, const al::IUseCamera*, float);
+
+void calcLayoutPosFromScreenPos(sead::Vector2f* output, const sead::Vector2f&);
+void calcLayoutPosFromWorldPos(sead::Vector2f* output, const al::IUseCamera*, const sead::Vector3f&);
+void calcLayoutPosFromWorldPos(sead::Vector3f* output, const al::IUseCamera*, const sead::Vector3f&);
+void calcLayoutPosFromWorldPos(sead::Vector2f* output, const al::SceneCameraInfo*, const sead::Vector3f&, int);
+void calcLayoutPosFromWorldPos(sead::Vector3f* output, const al::SceneCameraInfo*, const sead::Vector3f&, int);
+void calcLayoutPosFromWorldPosSub(sead::Vector2f* output, const al::IUseCamera*, const sead::Vector3f&);
+void calcLayoutPosFromWorldPosWithClampOutRange(sead::Vector3f* output, const al::IUseCamera*, const sead::Vector3f&, float, int);
+void calcLayoutPosFromWorldPosWithClampOutRange(sead::Vector3f* output, const al::SceneCameraInfo*, const sead::Vector3f&, float, int);
+void calcLayoutPosFromWorldPosWithClampByScreen(sead::Vector3f* output, const al::IUseCamera*, const sead::Vector3f&);
+float calcLayoutRadiusFromWorldRadius(const sead::Vector3f&, const al::IUseCamera*, float);
+
+bool calcCameraPosToWorldPosDirFromScreenPos(sead::Vector3f* output, const al::IUseCamera*, const sead::Vector2f&, float);
+bool calcCameraPosToWorldPosDirFromScreenPos(sead::Vector3f* output, const al::IUseCamera*, const sead::Vector2f&, const sead::Vector3f&);
+bool calcCameraPosToWorldPosDirFromScreenPos(sead::Vector3f* output, const al::SceneCameraInfo*, const sead::Vector2f&, float, int);
+bool calcCameraPosToWorldPosDirFromScreenPos(sead::Vector3f* output, const al::SceneCameraInfo*, const sead::Vector2f&, const sead::Vector3f&, int);
+void calcCameraPosToWorldPosDirFromScreenPosSub(sead::Vector3f* output, const al::IUseCamera*, const sead::Vector2f&, float);
+void calcCameraPosToWorldPosDirFromScreenPosSub(sead::Vector3f* output, const al::IUseCamera*, const sead::Vector2f&, const sead::Vector3f&);
+
+void calcLineCameraToWorldPosFromScreenPos(sead::Vector3f* output1, sead::Vector3f* output2, const al::IUseCamera*, const sead::Vector2f&);
```

Comment:
What's the difference between output1 and output2? These names could be clearer, or the distinction could be cleared up in a comment above these functions